### DISCUSSION TITLE
Daily 10/11

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#293] [General]
Ayer agregué metí todos los diagramas importantes dentro del directorio Documentación como imágenes por separado ya que en el pdf de diseño se veían chicos, también actualicé todos los pdfs para incorporar todo lo que hicimos hasta ahora, hoy voy a revisar a ver si me falta algo más en la documentación. No creo que sea necesario mover los directorios a front y back pero por si acaso preguntaré. También deployé el sitio con una github action para que se deploye automáticamente en dev o main. Ayer recibí todos los videos así que hoy me pongo a editarlos para crear uno solo y subirlo. También quiero dejar corriendo el servidor en AWS con MySQL. Y ayer metí a dependabot en el proyecto y me upgradeó varios de los módulos de python así que voy a volver a verificar que todo instale en una máquina de cero con las instrucciones que dejé en pdf.

En el sitio de Almoneda la experiencia todavía está incompleta, hoy la completo.

**Comment by AhumadaCeleste** [comment]
Ayer Estudie Inglés, rendí el último examen y se me corto la luz, por lo que que no pude completar la última lección para ya cerrar la materia. Hoy quisiera completar eso y retomar para finalizar Python. También probar de hacer algo en JavaScript. Espero lograrlo ya tengo un día un poco complejo con otras obligaciones. Mañana veré las clases de hoy de fullStack y SGC. y proseguiré con Python.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve estudiando Inglés y leyendo un poco de SGC.
Hoy rendí el exámen de Inglés y continuaré con la lectura de los materiales de SGC.  Voy a probar que el sitio en github funciene bien y tratare de hacer alguna prueba siguiendo las instrucciones del pdf. Por la tarde noche participaré en las clases sincrónicas de SGC y Full Stack.

**Comment by Brian-Almada** [comment]
Ayer estuve haciendo el portfolio para la pág de Almoneda. Hoy lo estuve retocando y haciendo algunas actualizaciones. Más luego voy a estar estudiando ingles para el examen.
